resources/idlharness.js

```diff
@@ -667,6 +676,10 @@ function exposed_in(globals) {
         self instanceof ServiceWorkerGlobalScope) {
         return globals.has("ServiceWorker");
     }
+    if (Object.getPrototypeOf(self) === Object.prototype) {
```

Comment:
Why not use `globalThis` here? The `self` property is set below, but why?

Reply:
idlharness predates `globalThis`, so it uses `self` everywhere. We could probably use `globalThis` everywhere now, but I didn't see a reason to start using it just here.